src/components/Photos/Photos.js

```diff
@@ -0,0 +1,33 @@
+import { connect } from 'react-redux';
+
+import {useFile} from '../../hooks/useFile'
+
+import './Photos.css';
+
+import heart from '../../assets/svg/heart-solid.svg';
+
+const Photos = (props) => {
+  const {files} = useFile()
+  return (
+    files.map((item) => {
+      return (
+        <article className="Article" key={item.id}>
+          <img className="Photo" src={item.url} alt={item.name} />
+          <p className="SingleLine">Place: {item.place}</p>
+          <p className="SingleLine">Date: {item.date}</p>
+          <p className="SingleLine">Tags: {item.tags}</p>
+          <img src={heart} className="Heart" alt="heart" />
+        </article>
+      );
+    })
+  );
+};
+
+
+const mapStateToProps = (state) => {
```

Comment:
O tym chciałabym porozmawiać na spotkaniu ;)

Reply:
Oki, tutaj jak wyżej, nie wiedziałam czy to zostawiać na później :)